utils/datasets.py

```
@@ -0,0 +1,29 @@
+def get_datasets() -> List[Dataset]:
+    return sorted(
+        map(
+            lambda filename: Dataset(
+                os.path.splitext(filename)[0],
+                pd.read_parquet(open(os.path.join(DATASETS_DIRECTORY, filename), "rb")),
+            ),
+            filter(
+                lambda filename: filename.endswith(".parquet"),
+                os.listdir(DATASETS_DIRECTORY),
+            ),
+        ),
```

Ruff

Unnecessary `map` usage (rewrite using a generator expression)

Replace `map` with a generator expression

(C417)